### DISCUSSION TITLE
How to see different architecture images in the Harbor UI?

### DISCUSSION THREAD

**Discussion by patrickdung** [goharbor/harbor#16000] [Q&A] [answered]
The manifest is multiarch (amd64 and arm64). The manifest and the image should be pushed into Harbor ("skopeo inspect --raw" confirms it). 
But the Web UI only show one item / artifact. It seems there's no indication of which platform/architecture is contained in that artifact.

Also, as an extension of this question, how do the scanners perform scanning on multi arch images? And how does harbor show the result of different arch? Sometimes the packages between two different arch is not exactly the same and bugs may occur on only one CPU architecture.

Thanks in advance!

**Comment by patrickdung** [comment, marked as answer]
Just discovered something... If it's OCI artifact, the artifact is clickable and you can see two items (amd64/arm64).
For Docker V2S2 it seems there is no indication about the architecture of the artifact in the table, but it is mentioned in the "Overview" section.

**Comment by 631068264** [comment]
https://github.com/goharbor/harbor/issues/6207#issuecomment-1370917423  not work for me
